src/en/space-station-14/mapping/guides/general-guide.md

```diff
@@ -103,21 +105,20 @@ To do this in game, find the ID of your station grid using F3 and hovering over
 This means you can have multiple stations loaded in a single map, so do with that what you will.
 
 ## Taking a screenshot of your station
-To screenshot your station you need both a map prototype, stored on Resources/Maps which is the prototype that the `savemap 2 name.yml` command from previous steps should generate and a game map prototype stored on Resources/Prototypes/Maps which contains the ID, path and other information, refer to the other prototypes in there as examples when making one.
+To screenshot your station you need both a map prototype, stored in `Resources/Maps` which is the prototype that the `savemap 1000 name.yml` command from previous steps should generate and a game map prototype stored on Resources/Prototypes/Maps which contains the ID, path and other information, refer to the other prototypes in there as examples when making one.
 
-After this, run the command `dotnet run --project Content.MapRenderer {your_map_id_here}`, with the game map prototype ID. You can also set command line arguments through your IDE and run it that way. The image should be inside Resources/MapImages once completed.
+After this, run the command `dotnet run --project Content.MapRenderer {your_map_id_here}` in your IDE or Git CLI, with the game map prototype ID (case-sensitive). You can also set command line arguments through your IDE and run it that way. The image should be inside Resources/MapImages once completed.
```

Comment:
If users do not know where to run a command, then it is also likely that their environments are not setup correctly. If including where, then include how or more specifically a link to how. 

~~2nt person~~